### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by daniel-panhead**
Also made navbar + footer responsive

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -4,13 +4,13 @@ import { RouterLink } from 'vue-router'
...
-    class="flex justify-between items-center drop-shadow-[0_-4px_3px_rgba(0,0,0,0.25)] bg-slate text-gray-50 py-1 px-3"
+    class="flex justify-between items-center drop-shadow-[0_-4px_3px_rgba(0,0,0,0.25)] bg-slate text-gray-50 py-6 lg:py-1 px-3"
   >
-    <a href="https://ubcbiomod.com" class="pl-2">
+    <a href="https://ubcbiomod.com" class="lg:pl-2 lg:text-lg text-center">
       See Past <span class="font-semibold">UBC Biomod</span> Projects
     </a>
 
-    <RouterLink to="/">
+    <RouterLink to="/" class="hidden lg:block">
```
Comment (IcedTea2K) [inline src/components/Footer.vue:13]: Why are we hiding the "Back to homepage" Logo for responsiveness? Are you going to disable this feature completely on mobile?

Comment (IcedTea2K) [reply src/components/Footer.vue:13]: Oh nvm, I didn't see that you add this to the navbar on mobile instead. Smart guy!

File: src/components/NavBar.vue [hunk 2]
```diff
@@ -20,4 +24,61 @@ import { RouterLink } from 'vue-router'
...
+    </RouterLink>
+    <button class="mr-10" @click="sidebar = !sidebar">
+      <svg class="active:opacity-70 transition-all duration-200 ease-in-out" viewBox="0 0 10 8" width="35">
+        <path d="M1 1h8M1 4h 8M1 7h8" class="stroke-gold" stroke-width="1.0" stroke-linecap="round" />
+      </svg>
+    </button>
+  </div>
+
+  <Transition>
+    <template v-if="sidebar">
```
Comment (IcedTea2K) [inline src/components/NavBar.vue:59]: Can you make the sidebar scrollable? Horizontal screen on mobile looks kinda wack, and you can't event access all the options.
image

File: src/components/NavBar.vue [hunk 3]
```diff
@@ -20,4 +24,61 @@ import { RouterLink } from 'vue-router'
...
+                  <RouterLink to="/team">Teams</RouterLink>
+                  <RouterLink to="#">Sponsors</RouterLink>
+              </div>
+          </div>
+        </div>
+    </template>
+  </Transition>
+
 </template>
+
```
Comment (IcedTea2K) [inline src/components/NavBar.vue]: For this, you could also add it as tailwindcss if you'd like.

File: src/App.vue [hunk 4]
```diff
@@ -1,7 +1,13 @@
 <script setup lang="ts">
 import { RouterView } from 'vue-router'
+import NavBar from '@/components/NavBar.vue'
+import Footer from '@/components/Footer.vue'
 </script>
 
 <template>
-  <RouterView />
+  <NavBar :key="$route.fullPath" />
```
Comment (IcedTea2K) [inline src/App.vue:8]: Just curious, why is it necessary to add the key here?

Comment (IcedTea2K) [reply src/App.vue:8]: Ohhhh, is it because there are multiple navbars, each for different page?

File: src/views/HomeView.vue [hunk 5]
```diff
@@ -1,39 +1,129 @@
...
+            <div class="w-80 h-80 bg-white"></div>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+              exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
```
Comment (IcedTea2K) [inline src/views/HomeView.vue:58]: Not sure if I like how this appears on mobile. The items seem to just go straight down, which looks kinda uggo. Is it possible to scale them to make them fit on one row? Or maybe add a carousel?

**Review by IcedTea2K — COMMENTED** [review]
Overall almost everything looks sexy, especially the navbar and footer. But there's some nitpick on my end :)

**Comment by IcedTea2K** [comment]
Also, your PR broke my carousel. :(

File: src/components/NavBar.vue (hunk 3 above)
Comment (daniel-panhead) [reply src/components/NavBar.vue]: Cannot - it's Vue animation transition classes that they use for animating components

Comment (IcedTea2K) [reply src/components/NavBar.vue]: I used it for the carousel brother.

File: src/components/NavBar.vue (hunk 2 above)
Comment (daniel-panhead) [reply src/components/NavBar.vue:59]: Not sure if my end result is the optimal look but it'll do for now I think

File: src/App.vue (hunk 4 above)
Comment (daniel-panhead) [reply src/App.vue:8]: The key forces the navbar to reload each time you navigate to a new page which resets the sidebar

**Review by IcedTea2K — APPROVED** [review]
Everything looks good now 👍

**Pull request merged**